assets/styles/views/partial/_header_carousel.scss

```diff
@@ -8,9 +8,11 @@
   }
 
   .overlay {
-    top:  calc(5vh + 210px) !important;
+    top:  calc(5vh + 225px) !important;
```

Comment:
let's refrain from changing heights and widths from now. let's focus on the fonts and typography.